### PR TITLE
Sdk metric parameter tracking for custom jobs

### PR DESCRIPTION
**Pull request by sudarshan-SpringML**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb [hunk 1]
```diff
@@ -8,7 +8,7 @@
       },
```
Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>Do not change the copyright year</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>Break the Overview .. Costs into separate (4) text cells.</p><p> </p><p>Move Dataset after Objective.</p><p> </p><p>Add Uses and Steps sections (see template) to Objective.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>The above two install cells should be combined into a single cell.</p><p> </p><p>The ! python3 -m pip -&gt; ! pip3</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>for PROJECT_ID, use the template version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>Cloud SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>use template version for BUCKET_URI and REGION</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>Redundant - remove cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>Add text cell above indicating downloading the data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <h3>Create a managed tabular dataset from a CSV -&gt;</h3><h3><br></h3><h3>Create a Vertex AI Tabular dataset from CSV data</h3><p> </p><p>A Managed dataset -&gt; A Vertex AI dataset</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>Each param on its own line, ex:</p><p> </p><p>result = aiplatform.method(</p><p>      param1=...,</p><p>      paramN=...</p><p>)</p><p>      </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>Run the following cell to create -&gt;</p><p> </p><p>Next, you create</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>Let's take out the commented lines on distributed training</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <h3>Vertex AI ML Metadata -&gt;</h3><h3>Vertex ML Metadata</h3>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>args=[ ...., ... ]</p><p> </p><p>should be</p><p> </p><p>args=[arg1,</p><p>           arg2,</p><p>           ...</p><p>          ]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>Title: Deploy Model -&gt; Deploy model</p><p> </p><p> </p><p>Deploy model to Google Cloud -&gt; </p><p>Next, deploy your Vertex AI Model resource to a Vertex AI Endpoint resource.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>These 2 text cells should be combined. Text cell needs a heading.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>text cell needs a heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

**Comment by sudarshan-SpringML** [comment]
Closing this PR due to merge conflicts. New PR is made for this file. We will follow this review comments, while making new PR. Thank you

**Pull request closed without merging**